### ISSUE TITLE
Latest QGIS Master crashes at start

### ISSUE THREAD

**Issue by dbobak** [qgis/QGIS#32662]
Latest QGIS Master crashes at start, generating the following report:

```
Crash ID: 5eca77f15c8e3b0b2ba93bfb4b5d5b41db0e1192

Stack Trace

QBasicMutex::unlockInternal :
QgsTask::~QgsTask qgstaskmanager.cpp:45
QgsNetworkContentFetcherTask::~QgsNetworkContentFetcherTask qgsnetworkcontentfetchertask.cpp:38
QgsNetworkContentFetcherTask::`vector deleting destructor' :
QObject::event :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify qgsapplication.cpp:418
QCoreApplication::notifyInternal2 :
QCoreApplicationPrivate::sendPostedEvents :
qt_plugin_query_metadata :
QEventDispatcherWin32::processEvents :
UserCallWinProcCheckWow :
DispatchMessageWorker :
QEventDispatcherWin32::processEvents :
qt_plugin_query_metadata :
QCoreApplication::processEvents :
QgisApp::QgisApp qgisapp.cpp:1228
main main.cpp:1344
WinMain mainwin.cpp:170
__scrt_common_main_seh exe_common.inl:253
BaseThreadInitThunk :
RtlUserThreadStart :

QGIS Info
QGIS Version: 3.11.0-Master
QGIS code revision: c86ed789bf
Compiled against Qt: 5.11.2
Running against Qt: 5.11.2
Compiled against GDAL: 3.1.0dev
Running against GDAL: 3.1.0dev

System Info
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.18362
```

**Comment by RandomRodder** [comment]
I have the same problem:
## User Feedback

Just updated to QGIS 3.10 yesterday and have not been able to open it since! Will not start. Crashes before it even opens. Hard to work when i can't open anything!

## Report Details

**Crash ID**: [5eca77f15c8e3b0b2ba93bfb4b5d5b41db0e1192]('https://github.com/qgis/QGIS/search?q=5eca77f15c8e3b0b2ba93bfb4b5d5b41db0e1192&type=Issues')


**Stack Trace**
<pre>
QBasicMutex::unlockInternal :
QgsTask::~QgsTask qgstaskmanager.cpp:45
QgsNetworkContentFetcherTask::~QgsNetworkContentFetcherTask qgsnetworkcontentfetchertask.cpp:38
QgsNetworkContentFetcherTask::`vector deleting destructor' :
QObject::event :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify qgsapplication.cpp:418
QCoreApplication::notifyInternal2 :
QCoreApplicationPrivate::sendPostedEvents :
qt_plugin_query_metadata :
QEventDispatcherWin32::processEvents :
UserCallWinProcCheckWow :
DispatchMessageWorker :
QEventDispatcherWin32::processEvents :
qt_plugin_query_metadata :
QCoreApplication::processEvents :
QgisApp::QgisApp qgisapp.cpp:1202
main main.cpp:1344
WinMain mainwin.cpp:170
__scrt_common_main_seh exe_common.inl:253
BaseThreadInitThunk :
RtlUserThreadStart :
</pre>


**QGIS Info**
QGIS Version: 3.10.0-A Coru�a
QGIS code revision: 4581d89fb3
Compiled against Qt: 5.11.2
Running against Qt: 5.11.2
Compiled against GDAL: 3.1.0dev
Running against GDAL: 3.1.0dev



**System Info**
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 6.1.7601

**Comment by dbobak** [comment]
I have QGIS 3.10 installed also. And the problem is only with Master, 3.10 works perfectly.

**Comment by roya0045** [comment]
@dbobak is your master up to date or slightly outdated?

**Comment by dbobak** [comment]
@roya0045, it is up to date. Updated yesterday. The problem appeared about a week ago and through daily updates it persists.

**Comment by dbobak** [comment]
@roya0045, I've updated just now (QGIS code revision | 1c1ceb1112) and it works now.

**Comment by roya0045** [comment]
@dbobak Glad to hear!
@RandomRodder I thought the issue was fixed before the 3.10 release, it should be fixed for 3.10.1 otherwise it might be another issue. You could also try the weekly build if those got fixed. Otherwise I would suggest making another issue ticket.
